### ISSUE TITLE
how to start contributing

### ISSUE THREAD

**Issue by CreativelyMe** [OfflineIMAP/imapfw#4]
Nicolas:

This isn't so much an issue as it is a question about how I might best offer assistance.   I didn't find an email address on your account or on the website, so I thought I'd start here. 

I have a long history of development, but I'm new to this specific area.  I'd really like to learn more about it and I generally learn best by digging in and doing some work.

Let me know

Joe (CreativelyMe)

**Comment by nicolas33** [comment]
Usually, the best way to get contact is to start with the [mailing list](http://lists.alioth.debian.org/mailman/listinfo/offlineimap-project) because both questions and answers get more audience. I'm willing to help you get started quickly via private mails if you prefer, though. I've updated my profile for the matter.

By chance, I wrote some materials recently (for the GSoC) that you might like to read:
- http://www.offlineimap.org/gsoc/
- http://www.offlineimap.org/gsoc/helpers.html

You're welcome.
